src/main/java/ca/bc/gov/chefs/etl/forms/pcd/fiscalYearReportingDates/processor/PcdFiscalYearReportingDatesApiResponseProcessor.java

```diff
@@ -0,0 +1,140 @@
+package ca.bc.gov.chefs.etl.forms.pcd.fiscalYearReportingDates.processor;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.camel.Exchange;
+import org.modelmapper.ModelMapper;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import ca.bc.gov.chefs.etl.constant.Constants;
+import ca.bc.gov.chefs.etl.constant.PCDConstants;
+import ca.bc.gov.chefs.etl.core.model.IModel;
+import ca.bc.gov.chefs.etl.core.model.SuccessResponse;
+import ca.bc.gov.chefs.etl.core.processor.BaseApiResponseProcessor;
+import ca.bc.gov.chefs.etl.forms.pcd.fiscalYearReportingDates.json.InterimReportingDates;
+import ca.bc.gov.chefs.etl.forms.pcd.fiscalYearReportingDates.json.PeriodReportingDates;
+import ca.bc.gov.chefs.etl.forms.pcd.fiscalYearReportingDates.json.QuarterReportingDates;
+import ca.bc.gov.chefs.etl.forms.pcd.fiscalYearReportingDates.json.Root;
+import ca.bc.gov.chefs.etl.forms.pcd.fiscalYearReportingDates.model.FiscalYearInterimDates;
+import ca.bc.gov.chefs.etl.forms.pcd.fiscalYearReportingDates.model.FiscalYearPeriodDates;
+import ca.bc.gov.chefs.etl.forms.pcd.fiscalYearReportingDates.model.FiscalYearQuarterDates;
+import ca.bc.gov.chefs.etl.forms.pcd.fiscalYearReportingDates.model.FiscalYearReportingDatesSubmission;
+import ca.bc.gov.chefs.etl.util.CSVUtil;
+import ca.bc.gov.chefs.etl.util.FileUtil;
+
+public class PcdFiscalYearReportingDatesApiResponseProcessor extends BaseApiResponseProcessor {
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public void process(Exchange exchange) throws Exception {
+        String payload = exchange.getIn().getBody(String.class);
```

Comment:
We have method in JsonUtil to handle non-ASCII characters but in this case I don't think it's necessary as they really shouldn't end up in this form and I'm fine with the job failing in that case. So just an FYI for future jobs.